tests/test_bot.py

```diff
@@ -0,0 +1,49 @@
+import time
+import pytest
+
+
+TIMEOUT = 60
+
+
+class TestAdminGroup:
+    def create_admingroup(self, admbot, botuser, db):
```

Comment:
maybe better to make this into an admingroup fixture to avoid the boilerplate in the three test functions.

Reply:
good point! I need the admbot and botuser objects, but I will just attach them to the Chat object the admingroup fixture provides.